src/languages/stan.js

```diff
@@ -523,18 +386,26 @@ export default function(hljs) {
         keywords: 'upper'
       },
       {
-        className: 'keyword',
-        begin: /\btarget\s*\+=/
+        // hack: in range constraints, upper must follow either , or <
+        // <multiplier = ..., offest = ...> or <offset = ...>
+        begin: /[<,]\s*offset\s*=/,
+        keywords: 'offset'
       },
       {
-        begin: '~\\s*(' + hljs.IDENT_RE + ')\\s*\\(',
-        keywords: DISTRIBUTIONS
```

Comment:
Why was this removed? Can distributions be used with out the trailing `(`?

Reply:
I wasn't sure what this was exactly doing. Distributions must end with the parenthesis so `example_dist();`

Reply:
Please ask vs just removing things you don't understand. :-)

This makes sure that distributions are ONLY highlighted when followed by a `(` which is good because it can prevent false positives - or conflicts with other languages that might sharing some of the same naming, but with a different syntax.  Generally the more precise we can match a thing, the better.

So I'd add this back and remove DISTRIBUTIONS from the generic `keywords` list...

Reply:
done